### PR TITLE
[ja] Sync docs/concepts/services-networking/ingress-controllers

### PR DESCRIPTION
**Pull request by ystkfujii**
Sync docs/concepts/services-networking/ingress-controllers

Edited Page on Deploy Preview: 
https://deploy-preview-40422--kubernetes-io-main-staging.netlify.app/ja/docs/concepts/services-networking/ingress-controllers/

### CODE REVIEW

**Review by t-inu — COMMENTED** [review]
I found some updated links compared to en page.
So, I left comments about these links as suggested changes.

**Comment by ystkfujii** [comment]
@t-inu 
Thanks for your review.
I updated about your comments. And additionally, I fixed the traefik link. https://github.com/kubernetes/website/pull/40422/commits/11bc549f3cc1d58aaa5f38a209bc3623ece4bf00

Could you please review again ?

File: content/ja/docs/concepts/services-networking/ingress-controllers.md [hunk 1]
```diff
@@ -21,24 +24,38 @@ Ingressリソースが動作するためには、クラスターでIngressコン
...
 * [Kong Ingress Controller for Kubernetes](https://github.com/Kong/kubernetes-ingress-controller#readme)は、[Kong Gateway](https://konghq.com/kong/)向けのIngressコントローラーです。
+* [Kusk Gateway](https://kusk.kubeshop.io/)は[Envoy](https://www.envoyproxy.io)をベースにしたOpenAPIドリブンのIngressコントローラーです。 
 * [NGINX Ingress Controller for Kubernetes](https://www.nginx.com/products/nginx/kubernetes-ingress-controller)は、[NGINX](https://www.nginx.com/resources/glossary/nginx/)ウェブサーバーで(プロキシとして)動作します。
+* [Pomerium Ingress Controller](https://www.pomerium.com/docs/k8s/ingress.html)は[Pomerium](https://pomerium.com/)ベースのものであり、コンテキストを考慮したアクセスポリシーを提供します。
 * [Skipper](https://opensource.zalando.com/skipper/kubernetes/ingress-controller/)は、カスタムプロキシーを構築するためのライブラリーとして設計された、Kubernetes Ingressなどのユースケースを含む、サービス構成用のHTTPルーターとリバースプロキシーです。
+* [Tyk Operator](https://github.com/TykTechnologies/tyk-operator)はAPI管理機能をIngressに持たせるためにCustom ResourcesでAPIを拡張します。Tyk OperatorはOpen Source Tyk GatewayとTyk Cloudコントロールプレーンで動作します。
 * [Traefik Kubernetes Ingress provider](https://doc.traefik.io/traefik/providers/kubernetes-ingress/)は、[Traefik](https://github.com/containous/traefik) proxy向けのIngressコントローラーです。
```
[the remark this answers is not in the document]
Comment (t-inu) [reply content/ja/docs/concepts/services-networking/ingress-controllers.md]: > https://github.com/containous/traefik -> https://github.com/traefik/

Sorry, I made a mistake.
https://github.com/containous/traefik -> https://traefik.io/traefik/
This is now fixed correctly.

**Review by t-inu — COMMENTED** [review]
/lgtm

**Comment by ystkfujii** [comment]
/assign @nasa9084

Could you review this?

File: content/ja/docs/concepts/services-networking/ingress-controllers.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 ---
 title: Ingressコントローラー
-reviewers:
+description: >-
+  クラスターで[Ingress](/ja/docs/concepts/services-networking/ingress/)を動作させるためには、_ingress controller_ が動作している必要があります。
+  少なくとも1つのIngressコントローラーを選択し、クラスター内にセットアップされていることを確認する必要があります。
+  このページはデプロイ可能な一般的なIngressコントローラーをリスト表示します。
```
Comment (nasa9084) [inline content/ja/docs/concepts/services-networking/ingress-controllers.md:6]:
```suggestion
  このページはデプロイ可能な一般的なIngressコントローラーをリストアップします。
```

**Comment by ystkfujii** [comment]
@nasa9084 

Thanks for your review. 
I updated about your comments. And I also did squash.

**Comment by ystkfujii** [comment]
@t-inu 

Would you add the LGTM label ?

**Review by t-inu — COMMENTED** [review]
/lgtm

**Review by nasa9084 — COMMENTED** [review]
/approve
sorry to be late~

**Pull request merged**